parm/ufs/mom6/MOM_input_template_500

```diff
@@ -258,6 +258,15 @@ Z_INIT_ALE_REMAPPING = True     !   [Boolean] default = False
                                 ! If True, then remap straight to model coordinate from file.
 
 ! === module MOM_diag_mediator ===
+NUM_DIAG_COORDS = 1
+                                ! The number of diagnostic vertical coordinates to use.
+                                ! For each coordinate, an entry in DIAG_COORDS must be provided.
+DIAG_COORDS = "z Z ZSTAR"
+                                ! A list of string tuples associating diag_table modules to
+                                ! a coordinate definition used for diagnostics. Each string
+                                ! is of the form "MODULE_SUFFIX,PARAMETER_SUFFIX,COORDINATE_NAME".
+DIAG_COORD_DEF_Z="FILE:@[MOM6_DIAG_COORD_DEF_Z_FILE],interfaces=zw"
+DIAG_MISVAL = @[MOM6_DIAG_MISVAL]
```

Comment:
Can these templates be updated in the UFS-weather-model.
In the GFS/global-workflow planning meeting, we discussed the need to use these templates from the ufs-weather-model.  Jun Wang spent significant amount of time in getting them synced and identified how the S2S without waves in the global-workflow does does not reproduce the configuration in the ufs-weather-model.

In that spirit, please consider syncing these changes in the templates in the ufs-weather-model.

Reply:
@aerorahul  I will hopefully be working on this some today.  I'll need to get @guillaumevernieres to help check things on Monday he'll be off this afternoon.  So timeline is likely early next week.

Reply:
Also need the 1-deg template (MOM_input_template_100) updated

Reply:
Done @jswhit ... I think. Can you double check?